mkosi

```diff
@@ -2965,8 +2966,8 @@ def load_args():
             die("Directory, subvolume and tar images cannot be booted.")
 
     if args.encrypt is not None:
-        if args.output_format not in (OutputFormat.raw_gpt, OutputFormat.raw_btrfs, OutputFormat.raw_squashfs):
-            die("Encryption is only supported for raw gpt, btrfs or squashfs images.")
+        if args.output_format not in RAW_FORMATS:
```

Comment:
Is RAW_FORMATS defines somewhere?

Reply:
Back on line [67](https://github.com/systemd/mkosi/pull/231/files#diff-6980c629516e4165bfaf96be8e5003cfR69)

Reply:
And now moved into the right commit.